static/js/webrtc.js

```diff
@@ -535,9 +596,23 @@ var rtc = (function() {
         // --use-fake-device-for-media-stream
         mediaConstraints.fake = true
       }
-      window.navigator.mediaDevices
+      return window.navigator.mediaDevices
```

Comment:
Return value only used in tests now. Taking advantage of waiting for the promise to finish.